README.md

```diff
@@ -16,3 +59,20 @@ in the inventory. For example, we create an `ethiopia_production` group that cov
 `ethiopia_production` inventory file.
 
 This allows us to centralize the configuration for each deployment target into a single location - `group_vars/<target>`
+
+### Vault ID labels
+
+Ansible supports the assignment of labels to different vault password files. It also allows the label (known as a vault
+ID) to be stored in the header of the encrypted vault file. This allows Ansible to automatically select a vault password
+when decrypting vault files. We use the vault ID feature to simplify vault management in this repository. The following
+touchpoints are involved:
+
+* `ansible.cfg`: A full list of vault IDs per deployment is declared in the Ansible config file
+* Vault passwords: Vault passwords will need to be stored in the developer's home directory with the appropriate
```

Comment:
Shall we put these as a how-to: How would I set up the vault password files to get up and running for an existing env?

Reply:
I've added this info to the existing "Editing vault secrets" for now, because I'm unsure how the getting started section will shake out. Once it starts forming, we can extract these bits into a more thorough "how-to".